nbs/directives.ipynb

```diff
@@ -0,0 +1,828 @@
+{
```

Comment:
<p>Great explanation.  You should also mention that naming your function or method with a <code>_</code> prefix does the same thing as well (also doesn't get placed in dunder all, not shown in docs etc)</p><p> </p><pre class="ql-syntax" spellcheck="false">#|export
def _private(): ...
</pre><p> </p><p>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>